coroner-client/src/main/java/backtraceio/coroner/query/CoronerValueFilter.java

```diff
@@ -0,0 +1,28 @@
+package backtraceio.coroner.query;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+
+public class CoronerValueFilter {
+    public final FilterOperator operator;
+    public final Object value;
+
+    public CoronerValueFilter(final FilterOperator operator, final Object value) {
+        this.operator = operator;
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return this.get().toString();
+    }
+
+    public JsonElement get() {
+        JsonArray result = new JsonArray();
+
+        result.add(operator.toString());
```

Comment:
wouldn't it be better if you pass string to constructor instead of an object? Why class implementation should know about converting operator to string?

Reply:
java thing...

Reply:
unfortunately it's a trick to have a string enum in Java